docs/changelogs/vNEXT.rst

```diff
@@ -23,14 +23,21 @@ Link to migration guide
 New features
 ============
 
-Feature 1
----------
-
-- summary
-- of
-- user facing changes
-
-Link to relevant documentation section
+``nil``
+-------
+
+Polar now pre-defines `a constant named ``nil``
```

Comment:
Tiniest of nits, but isn't it "predefined" and not hyphenated? Also, is it predefined or just defining?

Reply:
I think it's plain ole' defining

```suggestion
Polar now defines `a constant named ``nil``
```

Reply:
312bb23